app/src/main/java/in/ac/bits_pilani/goa/ard/models/Browser.java

```diff
@@ -0,0 +1,73 @@
+package in.ac.bits_pilani.goa.ard.models;
+
+import android.app.Activity;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsIntent;
+import android.webkit.URLUtil;
+import android.widget.Toast;
+
+import in.ac.bits_pilani.goa.ard.R;
+
+/**
+ * Browser opens web links.
+ * @author Rushikesh Jogdand
+ */
+class Browser {
+
+    /**
+     * Color of tab.
+     */
+    private static final int TAB_COLOR = R.color.colorPrimary;
+
+    /**
+     * CustomTabsIntent object.
+     */
+    private CustomTabsIntent customTabsIntent;
+
+    /**
+     * Host activity.
+     */
+    private final Activity host;
+
+
+    /**
+     * Constructor.
+     * @param host context of host activity.
+     */
+    Browser(final Activity host) {
+        this.host = host;
+    }
+
+    
+
+    /**
+     * Launches url in customized tab.
+     * @param url string url
+     *            must be a valid web url (i.e. "https://..." or "http://...")
+     */
+    void launchUrl(final String url) {
+        if (!URLUtil.isNetworkUrl(url)) {
+            host.runOnUiThread(new Runnable() {
+                @Override
+                public void run() {
+                    Toast.makeText(host, "Invalid URL", Toast.LENGTH_SHORT).show();
+                }
+            });
+            return;
+        }
+        init();
+        customTabsIntent.launchUrl(host, Uri.parse(url));
+    }
+
+    /**
+     * All customization done here.
+     */
+    private void init() {
+        final CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
+        customTabsIntent = builder.build();
+        //noinspection deprecation
+        builder.setToolbarColor(host.getResources().getColor(TAB_COLOR));
```

Comment:
don't use deprecated methods, I think you need to use 

`ContextCompat.getColor(host, R.color.whatever);`
again, the developer should be able to edit any part of the custom tab like this color field